site_scons/options.py

```diff
@@ -17,6 +17,22 @@
         help="""How many clonal families should we process per unseeded sample and locus? Defaults to 20,
         unless this is a test run in which case it defaults to 3.""")
 
+Script.AddOption('--only-seeds',
+        dest='only_seeds',
+        metavar='LIST',
+        default=None,
+        help="""Specify ':' separated list of partis seeds to process only those seed containing clusters.""")
+
+Script.AddOption('--match-indels-in-uid',
+        dest='match_indels_in_uid',
+        default=None,
+        help="""process only sequences matching the one indel in the sequence corresponding to the uid passed here in the annotation chosen in choose_cluster()""")
+
+Script.AddOption('--ignore-seed-indels',
```

Comment:
I'm _ok_ with this, however, I think that it would behoove us to also have this option on a case by case basis in the input yaml file. You might want to ignore seed indels in one particular cluster that you were familiar with, and run with match-indels-in-uid, but not have that apply towards _all_ of the seeds, where something unknown might slip through the cracks.